src/document.c

```diff
@@ -1191,7 +1191,7 @@ char_link(hoedown_buffer *ob, hoedown_document *doc, uint8_t *data, size_t offse
 			}
 			else if (data[i] == ')') {
 				if (nb_p == 0) break;
-				else nb_p--; i++;
+				else { nb_p--; i++; }
```

Comment:
Did you actually read the warning? This is a change in behavior is that what you intended?

Reply:
Contributions are generally intended to *improve* or at least in some way *help* a project. Are you being helpful?